### PR TITLE
GRANT-346 - create db typings

### PR DESCRIPTION
**Pull request by coryatwater-FW**
- update existing models
- update RO's
- update DTO's

### CODE REVIEW

File: client/pages/applications/[id]/score-table.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { useApplicationDetails } from 'services';
 
 const ScoreTable: NextPage = () => {
   const { query } = useRouter();
-  const { id } = query;
+  const id = query.id ? +query.id : undefined;
```
Comment (alva-fresh) [inline client/pages/applications/[id]/score-table.tsx]: I'm not sure here. Do you think might be better to add ? here: `query?.id`?
And remind me again, what's + for in `+query.id`?

File: client/constants/interfaces.ts [hunk 2]
```diff
@@ -96,7 +96,7 @@ export interface ApplicationTableProps {
   Created_At: string;
   Updated_At: string;
   Chefs_ID: string;
-  BCAAP_Form_ID: string;
+  BCAAP_Form_ID: number;
```
Comment (alva-fresh) [inline client/constants/interfaces.ts:99]: `BCAT` here?

File: client/pages/applications/[id]/score-table.tsx (hunk 1 above)
Comment (coryatwater-FW) [reply client/pages/applications/[id]/score-table.tsx]: it should handle it gracefully, but I can add it

`+` is shorthand for converting to number, `id` here is string by default

File: client/components/bcaap-form-sections/funding-eligibility.tsx [hunk 3]
```diff
@@ -134,19 +134,19 @@ export const FundingEligibility: React.FC = () => {
             <div className='flex flex-1 w-full justify-start'>
               <Radio
                 legend='The percentage share of BCAAP funding available for your project may be increased by an additional 5% if you are a non-profit society registered in British Columbia for the purposes of operating this small facility and you are reliant on volunteer contributions to complete this project. Does this description apply to you?'
```
Comment (alva-fresh) [inline client/components/bcaap-form-sections/funding-eligibility.tsx:136]: Maybe update legend here to remove BCAAP?
Also, the folder is called `bcaap-from-sections`.

File: client/constants/interfaces.ts (hunk 2 above)
Comment (alva-fresh) [reply client/constants/interfaces.ts:99]: Is this resolved?

**Review by alva-fresh — APPROVED** [review]

**Pull request merged**